client/client.go

```diff
@@ -11,7 +13,7 @@ type Client struct {
 	remoteClient pb.CasbinClient
 }
 
-func NewClient(ctx context.Context, address string, opts ...grpc.DialOption) (*Client, error) {
+func New(ctx context.Context, address string, opts ...grpc.DialOption) (*Client, error) {
```

Comment:
Still naming it as `NewClient` would be OK.

Reply:
Done